src/components/Products/Products.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { useState } from "react";
+import data from "./data";
+import { useCart } from "../chart/CartReducer";
+import { useDispatchCart } from "../chart/CartReducer";
+import { useSetState } from "../chart/CartReducer";
+
+import styles from "./Products.module.css";
+import * as giIcons from "react-icons/gi";
+import fullProdukt from "../chart/cart";
+
+function Products() {
+  const [click, setClick] = useState(false);
+
+  const clicked = () => setClick(!click);
+  const items = useCart();
+  const setState = useSetState();
+
+  const addToCart = (products) => {
+    let newCart = [...items];
+    let itemInCart = newCart.find((item) => item.name === products.name);
+
+    if (itemInCart) {
+      itemInCart.quantity++;
+    } else {
+      itemInCart = {
+        ...products,
+        quantity: 1,
+      };
+      newCart.push(itemInCart);
+    }
+    setState(newCart);
+  };
+  return (
+    <div>
+      <h1 className={styles.nadpis}> Produkty</h1>
+      <div className={styles.outer_wrapper}>
+        {data.map((item) => {
+          const { name, price, id, img } = item;
+          return (

```

Comment:
```suggestion
        {data.map(( { name, price, id, img }) => (
```

And then fix the brackets at end